libs/designer/src/lib/core/actions/bjsworkflow/initialize.ts

```diff
@@ -433,15 +439,26 @@ export const updateAllUpstreamNodes = (state: RootState, dispatch: Dispatch): vo
 };
 
 export const getCustomSwaggerIfNeeded = async (
+  operationId: string,
```

Comment:
INstead of passing this on, we can have a metadata inside customSwagger of manifest to specify whether this is apim or function, so the function getDynamicSwaggerUrl can do it based on that.
We have something similar for connection creation inside manifest which says the conection is apim or function